src/components/NewFeatureOverlay.vue

```diff
@@ -84,6 +85,12 @@ export default class NewFeatureOverlay extends Vue {
   @Prop({ default: "/" })
   buttonRoute!: string;
 
+  @Prop({ default : 'button'})
```

Comment:
nice
can we keep the default as 'router-link'? I see it being the most common use case